pymeasure/instruments/thorlabs/elliptec_utils/base.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please add the normal pymeasure copyright notice at the top instead of these docstrings.

Equally with the other files.